### PR TITLE
Packingslip

### PR DESCRIPTION
**Pull request by richpeck**
Further to my original PR, this fixes the aforementioned `packingSlipUrl=` issue

### CODE REVIEW

File: lib/pwinty/order.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@ class Order < Pwinty::Base
         attribute :stateOrCounty, Types::String.optional
         attribute :status, Types::String
         attribute :payment, Types::String
+        attribute :packingSlipUrl, Types::String.optional
```
Comment (tomharvey) [inline lib/pwinty/order.rb]: The `packingSlipUrl` is often absent from the response, which will cause an error to be raised when we declare that it should be present.

The suggestion below will allow it to be passed in - fixing the error you see - while also allowing it to be absent.

```suggestion
        attribute? :packingSlipUrl, Types::String.optional
```

If you implement this suggestion, you'll notice that the tests will pass and I will be happy to merge it in.

Comment (richpeck) [reply lib/pwinty/order.rb]: Rgr, will do it now

**Comment by richpeck** [comment]
👍

**Comment by tomharvey** [comment]
Massive thanks for your help in identifying and solving these bugs @richpeck - v3.0.4 isn now in ruby gems and should fix both of the issues you saw.

Let me know if any others crop up for you. I guess I have a limited use case and don't run into the packing slips or fancy paper options.

**Comment by richpeck** [comment]
That's fine, it's all on you for creating & maintaining the gem! I'm going to update my app to the latest version & see how it goes :) Thanks again for all the help.

**Pull request merged**